File: problems/1672_richest_customer_wealth/solution.cpp
#include <vector>
#include <string>
#include <unordered_map>
#include <unordered_set>
#include <algorithm>
#include <climits>
#include <queue>
#include <stack>
#include <cmath>
#include <numeric>
#include <set>
#include <map>
using namespace std;

class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int maxWealth = 0;
        for (auto& customer : accounts) {
            maxWealth = max(maxWealth, accumulate(customer.begin(), customer.end(), 0));
        }
        return maxWealth;
    }
};

File: problems/1672_richest_customer_wealth/Solution.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int max = 0;
        for (int[] customer : accounts) {
            int sum = 0;
            for (int account : customer) sum += account;
            max = Math.max(max, sum);
        }
        return max;
    }
}
